### DISCUSSION TITLE
Generated a map from yaml

### DISCUSSION THREAD

**Discussion by kpipk** [clj-commons/clj-yaml#85] [Q&A]
Hello,

I would like to take a yaml file and generate a map from it, using the the names and indentations to create the keys.  For example:
aws:
    region:  us-west-2

it would generate something like this:
(:aws-region "us-west-2"}

or some equivalent.

Why do I want to do this?  I'm trying to write a token replacer and not use yaml anchors.  I would like to take a value defined at the top of the yaml, and use its value down below somewhere in the code

**Comment by borkdude** [comment]
Can you maybe put the YAML code in between triple backticks? And what have you tried to so far? Was the output you for from clj-yaml not what you expected?

**Reply by kpipk** [reply]
Thanks for the quick reply.  I started using Anchors again, and I think that will work out.
